redis/src/cluster_async/mod.rs

```diff
@@ -226,36 +230,70 @@ impl<C> Dispose for ClusterConnInner<C> {
 }
 
 #[derive(Clone)]
-enum CommandRouting<C> {
-    Route(RoutingInfo),
+enum RoutingInfo<C> {
```

Comment:
and the enums in mod.rs as InternalRoutingInfo , InternalRoutingSingleNode, or something similar? something that would make the relationship between them clearer. 
We can also change the external routing from 'info' to 'type', e,g,
```
RoutingType- single/multi
SingleNodeRoutingType - random/specific
MultipleNodeRoutingType-all/primaries/..
```
but if it's already in redis-rs IDK if we want to break the API

Reply:
changed the enums in mod.rs to InternalRoutingInfo, InternalRoutingSingleNode.